src/Types/Value.purs

```diff
@@ -373,56 +478,63 @@ eq :: Value -> Value -> Boolean
 -- If both are zero then checkBinRel will be vacuously true, but this is fine.
 eq = checkBinRel (==)
 
+isAda :: CurrencySymbol -> TokenName -> Boolean
```

Comment:
As above, I think it would be clearer to add `unsafe` somewhere in here. I mean that we shouldn't refer to something that should never exist as "ada" (and I like the `unsafeAda...` convention you use elsewhere)

Reply:
This one I would agree with because we shouldn't be able to construct the adaSymbol etc.